.github/workflows/test_and_publish.yml

```diff
@@ -154,7 +158,7 @@ jobs:
         with:
           creds: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
           src: kedro-vertexai-e2e:latest
-          dst: gcr.io/gid-ml-ops-sandbox/kedro-vertexai-e2e:${{ github.sha }}
+          dst: gcr.io/gid-ml-ops-sandbox/kedro-vertexai-e2e:latest
```

Comment:
Do we worry that this can produce conflicts when multiple tests are running at the same time?